src/Microsoft.ML.Data/Transforms/ValueMapping.cs

```diff
@@ -753,26 +774,29 @@ private protected override void SaveModel(ModelSaveContext ctx)
         /// </summary>
         private abstract class ValueMap
         {
+            public readonly string KeyColumnName;
```

Comment:
>KeyColumnName [](start = 35, length = 13)

what do you think about using the entire column, rather than the name?

Reply:
I will merge `Name` and `Type` into `DataViewSchema.Column` and clean redundant functions up.

---
In reply to: [271579703](https://github.com/dotnet/machinelearning/pull/3180#discussion_r271579703) [](ancestors = 271579703)